.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v4
         with:
-          node-version: 20.17.0
+          node-version: 22
```

Comment:
Whether we know the production version in advance or the test version in advance, we should deploy what we test and test what we deploy.

Right now, production shows this:
```
$ node --version
v20.15.1
```

We pretty much don't know what version will come with the DO buildpack until we do a test deployment. If we accept this spirit of embracing uncertainty around which version we're running, then yeah, there is no point in trying to make it match, and 22 should be declared here. It is still too loosey-goosey for my taste.

Reply:
@bickelj appreciated all around, and I want to make sure you know I do see the value in what you're saying; I think it's worth leaning into that for now given the overall tradeoffs (and, as you say, the DO side of things too).